src/web/lib/ArticleRenderer.tsx

```diff
@@ -46,6 +49,20 @@ const commercialPosition = css`
     position: relative;
 `;
 
+const unsafeEmbedWrapperStyles = css`
+    position: relative;
+    float: left;
+    margin-top: 6px;
+    margin-bottom: 12px;
+    margin-right: 20px;
+    ${from.leftCol} {
+        margin-left: -100px;
```

Comment:
Are we sure we want to do this for every one? Also with the width below?

Reply:
I have had a look at the SCSS in frontend, it uses a bunch of calculations to determine width/margin etc...

in this case particularly : `margin-left: -1 * ($left-column + $gs-gutter);`


**Global variables**
`$gs-gutter: 20px;`
`$gs-column-width: 60px;`

**$left-column**: `gs-span(2)` = `$n-columns * $gs-column-width + $gs-gutter * ($n-columns - 1)` = `2 * 60px + 20px * (2 - 1)` = `140px`

**$gs-gutter**: `20px`

After having stepped through the SCSS logic, I think it is a lot simpler to just have the exact values defined rather than using the methods defined as the calculations will always resolve to the same numbers.